### PR TITLE
Udate to version 0.47

### PR DESCRIPTION
**Pull request by jorisvandenbossche**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: memory_profiler-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/m/memory_profiler/memory_profiler-{{ version }}.tar.gz
-  sha256: dce6e931c281662a500b142595517d095267216472c2926e5ec8edab89898d10
+  sha256: e992f2a341a5332dad1ad4a008eeac7cfe78c7ea4abdf7535a3e7e79093328cb
 
 build:
   number: 1
```
Comment (jakirkham) [inline recipe/meta.yaml]: Please reset the build number to 0.

**Comment by jakirkham** [comment]
One comment. Any interest in adding yourself as a maintainer?

**Comment by jakirkham** [comment]
Thanks @jorisvandenbossche.

**Comment by jorisvandenbossche** [comment]
> Any interest in adding yourself as a maintainer?

Well, I am not planning to actively watch memory_profiler for new releases (I just wanted to install it yesterday, and noticed that it was an old version on conda-forge), so not sure if that would be helpful (apart from being able to merge someone elses PR)

**Pull request merged**